File: tests/go/cases/case_249_alias_hop_chain_05.go
package escape_tests

var retainedCase249 = []map[string]string{}

func Case249AliasHopChain05(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "alias_hop_chain_05",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "alias_hop_chain_05:" + raw,
	}
	a := payload
	b := a
	c := b
	// ESCAPE: multi-hop alias chain retained globally.
	retainedCase249 = append(retainedCase249, c)
	return "ok"
}

File: tests/nodejs/cases/case_249_alias_hop_chain_05.js
/**
 * Task case 249: alias_hop_chain_05 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case249AliasHopChain05(input) {
  const taskName = 'alias_hop_chain_05';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const a = payload;
  const b = a;
  const c = b;
  // ESCAPE: multi-hop alias chain into retained sink.
  retainedAudit.push(c);
  return 'ok';
}

module.exports = { case249AliasHopChain05 };
